curvefs/src/client/fuse_client.cpp

```diff
@@ -1297,10 +1301,6 @@ CURVEFS_ERROR FuseClient::FuseOpRelease(fuse_req_t req, fuse_ino_t ino,
         return ret;
     }
 
-    if (enableSumInDir_) {
```

Comment:
why not delete

Reply:
> why not delete

Oplookup will not called when vfs cached the info of inode, so if delete the relationship when release maybe lost the parent.